### DISCUSSION TITLE
Angular Build is not creating.

### DISCUSSION THREAD

**Discussion by irfannasim** [nolimits4web/swiper#4088] [Q&A]
After integration of Swiper in my Angular9 Project it is working fine. but when i am creating the build it is giving error. i am unable to understand. please help to solve.

`ERROR in ./node_modules/swiper/angular/swiper_angular.ngfactory.js
Module not found: Error: Can't resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\angular'
resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\angular'
  Parsed request is a module
  using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\angular\package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in D:/D-Drive/Workspaces/GIT/Medicta/medicta-patient-fe
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: .)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\angular\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\node_modules doesn't exist or is not a directory
      D:\D-Drive\node_modules doesn't exist or is not a directory
      D:\node_modules doesn't exist or is not a directory
      looking for modules in D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules/swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\angular\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\node_modules]
[D:\D-Drive\Workspaces\GIT\node_modules]
[D:\D-Drive\Workspaces\node_modules]
[D:\D-Drive\node_modules]
[D:\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js]
 @ ./node_modules/swiper/angular/swiper_angular.ngfactory.js 8:0-37 10:50-65 10:398-413 10:415-430 28:239-257 29:73-91
 @ ./src/app/components/home/home.component.ngfactory.js
 @ ./src/app/components/home/home.module.ngfactory.js
 @ ./$$_lazy_route_resource lazy namespace object
 @ ./node_modules/@angular/core/fesm2015/core.js
 @ ./src/main.ts
 @ multi ./src/main.ts
ERROR in ./src/app/components/doctor-booking/doctor-booking.module.ngfactory.js
Module not found: Error: Can't resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking'
resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking'
  Parsed request is a module
  using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./src/app/components/doctor-booking)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in D:/D-Drive/Workspaces/GIT/Medicta/medicta-patient-fe
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: .)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\node_modules doesn't exist or is not a directory
      D:\D-Drive\node_modules doesn't exist or is not a directory
      D:\node_modules doesn't exist or is not a directory
      looking for modules in D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules/swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\node_modules]
[D:\D-Drive\Workspaces\GIT\node_modules]
[D:\D-Drive\Workspaces\node_modules]
[D:\D-Drive\node_modules]
[D:\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js]
 @ ./src/app/components/doctor-booking/doctor-booking.module.ngfactory.js 88:0-38 90:6569-6585 90:6587-6603
 @ ./$$_lazy_route_resource lazy namespace object
 @ ./node_modules/@angular/core/fesm2015/core.js
 @ ./src/main.ts
 @ multi ./src/main.ts
ERROR in ./src/app/components/doctor-booking/doctor-booking.component.ngfactory.js
Module not found: Error: Can't resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking'
resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking'
  Parsed request is a module
  using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./src/app/components/doctor-booking)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in D:/D-Drive/Workspaces/GIT/Medicta/medicta-patient-fe
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: .)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\node_modules doesn't exist or is not a directory
      D:\D-Drive\node_modules doesn't exist or is not a directory
      D:\node_modules doesn't exist or is not a directory
      looking for modules in D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules/swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\doctor-booking\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\node_modules]
[D:\D-Drive\Workspaces\GIT\node_modules]
[D:\D-Drive\Workspaces\node_modules]
[D:\D-Drive\node_modules]
[D:\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js]
 @ ./src/app/components/doctor-booking/doctor-booking.component.ngfactory.js 14:0-37 100:253-276 114:100-118
 @ ./src/app/components/doctor-booking/doctor-booking.module.ngfactory.js
 @ ./$$_lazy_route_resource lazy namespace object
 @ ./node_modules/@angular/core/fesm2015/core.js
 @ ./src/main.ts
 @ multi ./src/main.ts
ERROR in ./src/app/components/home/home.module.ngfactory.js
Module not found: Error: Can't resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home'
resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home'
  Parsed request is a module
  using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./src/app/components/home)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in D:/D-Drive/Workspaces/GIT/Medicta/medicta-patient-fe
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: .)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\node_modules doesn't exist or is not a directory
      D:\D-Drive\node_modules doesn't exist or is not a directory
      D:\node_modules doesn't exist or is not a directory
      looking for modules in D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules/swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\node_modules]
[D:\D-Drive\Workspaces\GIT\node_modules]
[D:\D-Drive\Workspaces\node_modules]
[D:\D-Drive\node_modules]
[D:\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js]
 @ ./src/app/components/home/home.module.ngfactory.js 89:0-38 91:6696-6712 91:6714-6730
 @ ./$$_lazy_route_resource lazy namespace object
 @ ./node_modules/@angular/core/fesm2015/core.js
 @ ./src/main.ts
 @ multi ./src/main.ts
ERROR in ./src/app/components/home/home.component.ngfactory.js
Module not found: Error: Can't resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home'
resolve 'swiper_angular' in 'D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home'
  Parsed request is a module
  using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./src/app/components/home)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in D:/D-Drive/Workspaces/GIT/Medicta/medicta-patient-fe
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: .)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular doesn't exist
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\Medicta\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\GIT\node_modules doesn't exist or is not a directory
      D:\D-Drive\Workspaces\node_modules doesn't exist or is not a directory
      D:\D-Drive\node_modules doesn't exist or is not a directory
      D:\node_modules doesn't exist or is not a directory
      looking for modules in D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules
        using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
          using description file: D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\package.json (relative path: ./node_modules/swiper_angular)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx doesn't exist
            .mjs
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js doesn't exist
            as directory
              D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular doesn't exist
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\swiper_angular.js]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\home\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\components\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\app\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\src\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\node_modules]
[D:\D-Drive\Workspaces\GIT\node_modules]
[D:\D-Drive\Workspaces\node_modules]
[D:\D-Drive\node_modules]
[D:\node_modules]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.ts]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.tsx]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.mjs]
[D:\D-Drive\Workspaces\GIT\Medicta\medicta-patient-fe\node_modules\swiper_angular.js]
 @ ./src/app/components/home/home.component.ngfactory.js 12:0-37 69:235-258 83:100-118
 @ ./src/app/components/home/home.module.ngfactory.js
 @ ./$$_lazy_route_resource lazy namespace object
 @ ./node_modules/@angular/core/fesm2015/core.js
 @ ./src/main.ts
 @ multi ./src/main.ts
`

**Comment by vltansky** [comment]
Hey, can you reproduce the bug on stackblitz / public repo ?

**Comment by wnabil** [comment]
I have the same issue but cant reproduce on stackblitz 
any idea what could cause the issue please ? @vltansky

**Reply by vltansky** [reply]
Can you share repo ?

**Reply by wnabil** [reply]
> Can you share repo ?

Sadly i cant its a private project, i can share the package.json file and the system specs if that's could help

**Reply by vltansky** [reply]
Just make a new project and try to reproduce the issue, so you can share it here.

**Reply by vltansky** [reply]
and yes `package.json` and system specs would be useful too
